lib/core/ui/root_widget.dart

```diff
@@ -34,57 +36,53 @@ Widget injectUMEWidget({
   enable
       ? PluggableMessageService().resetListener()
       : PluggableMessageService().clearListener();
-  return _FloatingWidget(
-      child: child,
-      enable: enable,
-      supportedLocales: supportedLocales,
-      localizationsDelegates: localizationsDelegates);
+  WidgetsFlutterBinding.ensureInitialized();
+  WidgetsBinding.instance?.addPostFrameCallback((timeStamp) {
+    if (enable) {
+      final overlayEntry = OverlayEntry(builder: (BuildContext context) {
+        return _FloatingWidget();
+      });
```

Comment:
It will be better if the `_FloatingWidget` has a constant constructor. (`const _FloatingWidget`)

```suggestion
      final OverlayEntry overlayEntry = OverlayEntry(
        builder: (_) => const _FloatingWidget(),
      );
```